model/job-management/src/main/proto/beam_artifact_api.proto

```diff
@@ -102,13 +99,19 @@ message ArtifactChunk {
   bytes data = 1;
 }
 
+message PutArtifactMetadata {
+  // (Required) A token for artifact staging session.
```

Comment:
Are all PutArtifactMetadata's within the same PutArtifactRequest stream required to use the same token?

Also: document where this token is supposed to come from?